docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+---
+sidebar_position: 2
+slug: /modeling/migrating/migrating-schema-1-1
+---
+
+import {
+  AuthzModelSnippetViewer,
+  CardBox,
+  CheckRequestViewer,
+  DocumentationNotice,
+  ProductConcept,
+  ProductName,
+  ProductNameFormat,
+  ReadRequestViewer,
+  RelatedSection,
+  RelationshipTuplesViewer,
+  WriteRequestViewer,
+} from '@components/Docs';
+
+# Migrating Models To Schema 1.1
+
+<DocumentationNotice />
+
+<ProductName format={ProductNameFormat.ShortForm} /> has introduced a new DSL schema version with several changes that we
+believe will make models easier to read and write, enable better tuple and model validations, and provide more options for
+optimizing the performance of different <ProductName format={ProductNameFormat.ShortForm} /> APIs.
+
+In short, we will be:
+
+1. [Adding model schema version field](#-model-schema-versions)
```

Comment:
When I click on these links they overshoot the link header by a large margin.

Reply:
But I think it is more as a by-product of you using `npm run dev` vs `npm run build; npm run serve`